src/modules/market/components/market-positions-list--order/market-positions-list--order.jsx

```diff
@@ -76,7 +80,11 @@ export default class Order extends Component {
           <li />
         }
         <li>
-          <button onClick={this.toggleConfirm}>Cancel</button>
+          { p.pending === CLOSE_DIALOG_CLOSING ?
+            <span className={Styles.NotActive}>Cancel</span>
+            :
```

Comment:
I keep meaning to codify this into a listing rule, but put the `:` at the end of the first expression.